src/internal/_stepCat.js

```diff
@@ -0,0 +1,54 @@
+var _add = require('./_add');
+var _appendTo = require('./_appendTo');
+var _isTransformer = require('./_isTransformer');
+var _symTransformer = require('./_symTransformer');
+var isArrayLike = require('../isArrayLike');
+var merge = require('../merge');
+
+
+module.exports = (function() {
+  var _result = function(result) { return result; };
+  var _stepCatArray = {
+       init: function() { return []; },
+       step: _appendTo,
+       result: _result
+  };
+  var _stepCatString = {
+      init: function() { return ''; },
+      step: _add,
+      result: _result
+  };
+  var _stepCatObject = {
+      init: function() { return {}; },
```

Comment:
Yepz as long as the user isn't going to be calling them directly
On Mar 10, 2015 6:35 PM, "Kevin Beaty" <notifications@github.com> wrote:

> In src/internal/_stepCat.js
> <https://github.com/ramda/ramda/pull/878#discussion_r26173653>:
>
> > +
> > +
> > +module.exports = (function() {
> > +  var _result = function(result) { return result; };
> > +  var _stepCatArray = {
> > +       init: function() { return []; },
> > +       step: _appendTo,
> > +       result: _result
> > +  };
> > +  var _stepCatString = {
> > +      init: function() { return ''; },
> > +      step: _add,
> > +      result: _result
> > +  };
> > +  var _stepCatObject = {
> > +      init: function() { return {}; },
>
> Interesting. String fine as well, I presume?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ramda/ramda/pull/878/files#r26173653>.
>